frontend-react/src/pages/daily/Table/ReportsUtils.ts

```diff
@@ -0,0 +1,42 @@
+import download from "downloadjs";
+import axios from "axios";
+
+import ReportResource from "../../../resources/ReportResource";
+
+export const reportDetailURL = (id: string, base?: string) =>
+    `${
+        base ? base : process.env.REACT_APP_BACKEND_URL
+    }/api/history/report/${id}`;
+
+export const getReportAndDownload = (
+    id: string,
+    oktaToken: string,
```

Comment:
I could not get unit tests for this to work. I tried mocking `axios.get` to return a report and `download` from downloadjs to just an empty `jest.fn()`. When expecting the calls, neither would get called for some reason.